tests/key_field/key_field_sample_generated.h

```diff
@@ -67,11 +67,11 @@ FLATBUFFERS_MANUALLY_ALIGNED_STRUCT(1) Baz FLATBUFFERS_FINAL_CLASS {
   bool KeyCompareLessThan(const Baz * const o) const {
     return KeyCompareWithValue(o->a()) < 0;
   }
-  int KeyCompareWithValue(const flatbuffers::Array<uint8_t, 4> *_a) const { 
+  int KeyCompareWithValue(const flatbuffers::Array<uint8_t, 4> *_a) const {
     for (auto i = 0; i < a()->size(); i++) {
-      const auto a_l = a_[i];
-      const auto a_r = _a->Get(i);
-      if(a_l != a_r) 
+      const auto a_l = flatbuffers::EndianScalar(a_[i]);
+      const auto a_r = flatbuffers::EndianScalar(_a->Get(i));
```

Comment:
this is doing the swap twice, since the array->Get() call does the swap internally